core/tests/performance_tests/base.py

```diff
@@ -47,20 +67,23 @@ def _load_page_to_cache_server_resources(self):
 
     def _record_page_metrics_for_url(self):
         self.page_metrics = (
-            self.data_fetcher.get_page_metrics_for_url(self.page_url))
+            self.data_fetcher.get_page_metrics_for_url(self.page_url,
```

Comment:
Nit: please use consistent formatting. You should do a line break at ``(`` and then the arguments can stay on the same line.